### PR TITLE
watchdog insights

### PR DESCRIPTION
**Pull request by HugoPct**
filtering on a outlier makes it disapear

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback, but otherwise looks good.

File: content/en/watchdog/insights.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ Every outlier comes with embedded interactions and a side panel with troubleshoo
 ### Filter on Insight query
 
 To refine your current view to match a Watchdog Insight, hover over the top right corner of an Insight summary card. Two icons appear. Click on the inverted triangle icon with the tooltip **Filter on Insight**. The page refreshes to show a list of entries corresponding to the insight.
+Beware, fIltering on Watchdog Insights will automatically change the scope you're looking at. Hence, if you select an outlier insight, it won't be shown anymore but the traces and scope associated.
```
[the remark this answers is not in the document]
Comment (HugoPct) [reply content/en/watchdog/insights.md:60]: when you filter on an insights outlier, the scope of the query is updated. 
As you are scoped on the traces of the outlier, the outlier is not an outlier anymore but the baseline, therefore we do not show it.
what do you think about the latest suggestion?

[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/watchdog/insights.md:60]: That's help, thanks @HugoPct. I updated my suggestion. What do you think?

[the remark this answers is not in the document]
Comment (HugoPct) [reply content/en/watchdog/insights.md:60]: @rtrieu 
great!
can we be a bit more specific and explicitly say the outlier will not be visible anymore

[the remark this answers is not in the document]
Comment (rtrieu) [reply content/en/watchdog/insights.md:60]: @HugoPct yes!

[the remark this answers is not in the document]
Comment (HugoPct) [reply content/en/watchdog/insights.md:60]: @rtrieu 
thanks!
looks good to me

**Pull request merged**